src/vue-wrapper.ts

```diff
@@ -125,6 +133,25 @@ export class VueWrapper<T extends ComponentPublicInstance>
     return createWrapper(null, result[0])
   }
 
+  getComponent<T extends ComponentPublicInstance>(
+    selector: new () => T
+  ): VueWrapper<T>
+  getComponent<T extends ComponentPublicInstance>(
+    selector: FindComponentSelector
+  ): VueWrapper<T>
+  getComponent<T extends ComponentPublicInstance>(
+    selector: any
+  ): VueWrapper<T> {
+    const result = this.findComponent(selector)
+    if (result instanceof ErrorWrapper) {
+      throw new Error(
+        `Unable to find component with selector ${selector} within: ${this.html()}`
```

Comment:
This is really a small nitpick, but got bitten by it the other day 😅 the word "find" in the error message of "get" tricked me for a bit

```suggestion
        `Unable to get component with selector ${selector} within: ${this.html()}`
```

Reply:
Good point 🙂 And I think we should change the message for `get` as well to be coherent.